06-building-a-program-start-to-finish/solving-the-problem/Output/inverted-index.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Inverted Index class
+ */
+class InvertedIndex {
+/**
+ * class constructor
+ */
+  constructor() {
+    this.indices = {};
+    this.documentCount = 0;
+  }
+/**
+ * normalizedText takes a string with numbers,
+ * spaces and symbols and returns a normalized string
+ * @param{String} text - The name of the string to be normalized
+ * @return{String} normalized string
+ */
+  normalizedText(text) {
```

Comment:
Want to think of renaming this function? The tense seems odd.

Maybe `normalizeText`? :simple_smile: